e2e/rdctl.e2e.spec.ts

```diff
@@ -787,13 +790,74 @@ test.describe('Command server', () => {
           expect(stderr).toEqual('');
           expect(JSON.parse(stdout)).toEqual([
             'GET /v0',
+            'GET /v0/diagnostic_categories',
+            'GET /v0/diagnostic_check',
+            'GET /v0/diagnostic_ids',
             'PUT /v0/factory_reset',
             'PUT /v0/propose_settings',
             'GET /v0/settings',
             'PUT /v0/settings',
             'PUT /v0/shutdown',
           ]);
         });
+        test('/v1 should fail', async() => {
+          const { stdout, stderr } = await rdctl(['api', '/v1']);
+
+          expect({ stdout: JSON.parse(stdout), stderr: stderr.trim() }).toMatchObject({ stdout: { message: '404 Not Found' }, stderr: 'Unknown command: GET /v1' });
+        });
+
+        test.describe('diagnostics', () => {
+          let categories: string[];
+
+          test('categories', async() => {
+            const { stdout, stderr } = await rdctl(['api', '/v0/diagnostic_categories']);
+
+            expect(stderr).toEqual('');
+            categories = JSON.parse(stdout);
+            expect(categories).toMatchObject(['Utilities', 'Networking']);
```

Comment:
Same as the other test — `toMatchObject` doesn't do order-insensitive list comparison.

(I'll skip mentioning the other instances of the same issue.)

Reply:
Right, I forgot about these tests.